### DISCUSSION TITLE
Visualizar xplot_map

### DISCUSSION THREAD

**Discussion by JGranda11** [rramosp/ai4eng.v1#311] [05 - GESTIÓN DE MODELOS]
En la ejecución del Notebook "05.02 - MODEL EVALUATION", hay un problema cuando se hace uso de la librería Bokeh, ya que la celda que me permite visualizar el mapa con los parámetros que se dan en la función, me arroja un error:
![image](https://github.com/rramosp/ai4eng.v1/assets/112766584/e5113b65-0b3e-4dec-a41e-c9eae9a3686b)

**Comment by Felipe-RA** [comment]
Saludos @JGranda11 

Primero, gracias por el aviso. El error esta en un cambio de bokeh, especificamente en los nombres de los atributos

En lo que empujamos los cambios y se propagan, si quieres puedes solucionarlo fácilmente  si cambias `plot_width` and `plot height` por `min_width` y `min_height` respectivamente.

De nuevo, ¡gracias por el heads up!

**EDIT:** FIXED LAST WEEK WITH 3a5c67a
